editor/components/block-settings-menu/reusable-block-toggle.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { getBlock } from '../../selectors';
+import { convertBlockToStatic, convertBlockToReusable } from '../../actions';
+
+export function ReusableBlockToggle( { block, convertToStatic, convertToReusable } ) {
+	const isReusableBlock = block.name === 'core/block';
```

Comment:
Thinking purely from the [presentational / container distinction](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0), I might expect this variable to be passed as the prop, determined in a selector or `mapStateToProps`, particularly since we're not otherwise using the `block` prop.

Reply:
👍 I've moved it in 95a28a37adea3afeb44fb2b9c068c9e1fc61fb31.